client/dist/index.html

```diff
@@ -2,10 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
```

Comment:
This looks good. I'm just thinking out loud here -- and I'm wondering if using a CDN to download CSS will slow down our page loading time, which is not an issue now, but could become an issue next week. I remember Joseph mentioned something about tree shaking and only importing the parts of a library that we need. I'm also wondering if there might be a way to download the css files locally and then use css-loader (i.e. webpack) to serve them w/o any requests from client -> CDN. 

I'm curious to hear your thoughts

Reply:
These are great comments, thank you Aaron. 
Per your comments, I am thinking I will uninstall the vanilla dependency, get rid of the CDNs, install react-bootstrap, and import as needed in individual files. What do you think of that solution?

Reply:
After opening up the Front-End Optimization slides again I was reminded that the opening slide says "Don't do it. Optimize for dev time instead." I think you're right to go with the CDN option. I'm going to go ahead and approve your PR